### DISCUSSION TITLE
Get certain information from the results

### DISCUSSION THREAD

**Discussion by AshleyJTM** [theOehrly/Fast-F1#91] [Q&A] [answered]
Hi there, just a quick question. 

Is there a way to only display some of this information? / Other methods to gather information from the API. 

Currently race.results[0] displays all the information for the driver who won the race, I would like to tidy up this data or make is more specific. 

**This is my code:**
 if answer:
                year,gp = params[1].split(' ', 1)
                race = ff1.get_session(year, gp, 'R')
                pprint(race.results[0])
                print(params[1])
            else:
                print('Incorrect details, try again.')

This is the current output, but I'd like to narrow it down to just the driver name and some other details like nationality etc.

![image](https://user-images.githubusercontent.com/58674090/149383409-58841566-1941-421c-be47-f4ffe498a5eb.png)

Any help would be hugely appreciated, thank you.

**Comment by theOehrly** [comment, marked as answer]
Each item in `Session.results` is a dictionary. Therefore, you can just use the usual way of getting values from a dictionary.
For example, to get the driver code: `race.results[0]['Driver']['code']`
